primitives/src/asset.rs

```diff
@@ -0,0 +1,31 @@
+use crate::AssetId;
+
+use codec::{Decode, Encode};
+
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+
+/// Asset Pair representation
+/// ( asset_a, asset_b ) combination where asset_a is meant to be exchanged for asset_b
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Debug, Encode, Decode, Copy, Clone, PartialEq, Eq, Default)]
+pub struct AssetPair {
+	pub asset_in: AssetId,
```

Comment:
Maybe we should specify that this is asset pair in case of trades

Reply:
added bit more detailed explanation